libs/core/src/lib/token/tokenizer.component.ts

```diff
@@ -229,16 +245,39 @@ export class TokenizerComponent implements AfterViewChecked, AfterContentInit, O
         }
     }
 
+    /** @hidden */
+    @HostListener('document: click', ['$event'])
+    clickout(event) {
+        if (this.elementRef().nativeElement.contains(event.target) === false) {
+            this.tokenList.forEach(token => {
+                token.selected = false;
+            });
+        }
+    }
```

Comment:
I remember having some document listeners on popover. It was very bad for performance, for more than 100 popovers. I hope no one will put more than 10 tokenizers in 1 page.